about.html

```diff
@@ -0,0 +1,160 @@
+                    Event</h3>
+                <a href="#">altertime2022@classhall.org</a>
+            </div>
+        </header>
+    </section>
+    <!-- about Header Section starts -->
+
+    <!-- about logo Section Starts -->
+    <section id="aboutLogo">
+        <div class="aboutLogo__wrapper">
+            <h2 class="logoTitle">Altered Time Mages' Class Hall 2022 Logo</h2>
+            <div class="underline noShow"></div>
+            <p class="logoPara__mobile">Lorem ipsum dolor sit amet consectetur, adipisicing elit. Consequuntur at iure
+                nam recusandae molestiae ratione.</p>
+            <p class="logoPara__desktop">Lorem ipsum dolor sit amet consectetur adipisicing elit. Quisquam praesentium
+                quia rem dolorum officia! Deleniti unde culpa sint sit libero voluptates incidunt accusantium dolor?
+                Vitae? Lorem ipsum dolor sit, amet consectetur adipisicing elit. Quisquam reiciendis totam molestias?
+            </p>
+            <figure class="logoBlock">
+                <img src="img/mage icon.png" alt="Mage logo">
+            </figure>
+        </div>
+    </section>
+    <!-- about logo Section Ends -->
+
+    <!-- History Section Starts -->
+    <section id="past">
+        <div class="past__wrapper">
+            <h2>See the past Altertime Events</h2>
+            <div class="underline noShow"></div>
+            <p>Take a look at last two Altertime events which took place in Buenos Aires and in Warsaw.</p>
+            <div class="figures">
+                <figure class="figure1"><a href="#">
+                        <h3>2020</h3>
+                        <h5>Altertime event 2020 in Buenos Aires</h5>
+                    </a></figure>
+                <figure class="figure2"><a href="#">
+                        <h3>2021</h3>
+                        <h5>Altertime event 2021 in Warsaw</h5>
+                    </a></figure>
+            </div>
+        </div>
+    </section>
+    <!-- History Section Ends -->
+
+    <!-- Partner Section Starts Here -->
+    <section id="partner">
+        <h3 class="partnerTitle">Partner</h3>
+        <div class="partnerIcons">
+            <a href="https://www.adidas.com/us" target="_blank" rel="noopener noreferrer"><img src="img/partner/adidas-svgrepo-com.svg" alt="Adidas"></a>
+            <a href="https://www.airbnb.com/" target="_blank" rel="noopener noreferrer"><img src="img/partner/airbnb-2-logo-svgrepo-com.svg" alt="AirBnb"></a>
+            <a href="https://www.blizzard.com/en-us/" target="_blank" rel="noopener noreferrer"><img src="img/partner/blizzard-svgrepo-com.svg" alt="Blizzard"></a>
+            <a href="https://www.corsair.com/ww/en/" target="_blank" rel="noopener noreferrer"><img src="img/partner/corsair-svgrepo-com.svg" alt="Corsair"></a>
+            <a href="https://www.google.com/" target="_blank" rel="noopener noreferrer"><img src="img/partner/google-svgrepo-com.svg" alt="Google"></a>
+        </div>
+    </section>
```

Comment:
- [ ] Great job using flex to arrange the partnership brands! 💯 However, it could be improved to match the given design:

|Template|Your design|
|---|---|
| ![image](https://user-images.githubusercontent.com/77069011/204030883-dac617ae-b7d0-45cb-b938-786d170bbd69.png) | ![image](https://user-images.githubusercontent.com/77069011/204030819-4224df90-2ebb-4d96-a899-f861ee6e9b63.png) |


💡 _Hint: You could define your `justify-content` property of the partner section container to `center`, and play a little with the width of each logo._

You could achieve this:
![image](https://user-images.githubusercontent.com/77069011/204031089-b2e0c70a-e3e5-44d9-beda-0e5d8405313a.png)